### PR TITLE
add calendar icon

### LINKED ISSUE #111: Add Calendar Icon
x-team/auto#748 uses a calendar icon for date input. Add a calendar to the SvgIcon component

### PR DESCRIPTION
**Pull request by jepz20**
Resolves #111 

captura de pantalla 2018-01-26 a la s 12 29 05 a m

### CODE REVIEW

File: src/components/ui/SvgIcon.js [hunk 1]
```diff
@@ -185,6 +185,23 @@ const getIcon = ({ icon, color }) => {
           </g>
         </g>
       </svg>
+    ),
+
+    calendar: (
+      <svg width='40px' height='40px' viewBox='0 0 16 16'>
+        <g stroke='none' strokeWidth='1' fill='none' fillRule='evenodd'>
+          <g transform='translate(-560.000000, -479.000000)' fill='#C7C7CA'>
```
Comment (bernardodiasc) [inline src/components/ui/SvgIcon.js]: Is it expected to leave the Hex value here or should be the case to be replaced by `colors[color]`?

Comment (jepz20) [reply src/components/ui/SvgIcon.js]: Fixed it, I was using the fill in the path but it was redundant so I move it here. thanks 😄

**Comment by jepz20** [comment]
Hey, @bernardodiasc I've updated the PR, could you check it again please 🙂

**Comment by nicksp** [comment]
👍  Just updated outdated spect and merging now 😄

**Pull request merged**